build_posix/configure.ac.in

```diff
@@ -31,7 +31,7 @@ LT_INIT([pic-only])
 AC_SUBST([LIBTOOL_DEPS])
 
 AC_PROG_CC(cc gcc)
-# AC_PROG_CXX(c++ g++)
+AC_PROG_CXX(c++ g++)
```

Comment:
If I do not make that change, I get the following error while running `autogen.sh`
> api/leveldb/Makefile.am: error: C++ source seen but 'CXX' is undefined
> api/leveldb/Makefile.am:   The usual way to define 'CXX' is to add 'AC_PROG_CXX'
> api/leveldb/Makefile.am:   to 'configure.ac' and run 'autoconf' again.